code/modules/jobs/job_types/enclave.dm

```diff
@@ -387,7 +416,44 @@
 	ADD_TRAIT(H, TRAIT_ENCLAVE_CODES, src)
 	H.grant_language(/datum/language/codespeak, TRUE, TRUE, LANGUAGE_MIND)
 
+
+/datum/outfit/loadout/enclave_marksman
+	name = "Marksman"
+	suit_store = /obj/item/gun/ballistic/automatic/marksman
+	backpack_contents = list(
+		/obj/item/ammo_box/magazine/m556/rifle = 2,
+		/obj/item/gun/ballistic/automatic/pistol/n99 = 1,
+		/obj/item/clothing/under/f13/ncr_camo = 1,
+		/obj/item/clothing/suit/armor/f13/metalarmor/steelbib = 1,
+		/obj/item/clothing/head/helmet/armyhelmet = 1,
+	)
+
+/datum/outfit/loadout/enclave_frontline
+	name = "Rifleman"
+	suit_store = /obj/item/gun/ballistic/automatic/service/r82
+	backpack_contents = list(
+		/obj/item/ammo_box/magazine/m556/rifle = 2,
+		/obj/item/gun/ballistic/automatic/pistol/n99 = 1,
+		/obj/item/clothing/under/f13/ncr_camo = 1,
+		/obj/item/clothing/suit/armor/f13/metalarmor/steelbib = 1,
+		/obj/item/clothing/head/helmet/armyhelmet = 1,
+	)
+
+
+/datum/outfit/loadout/enclave_shotgunner
+	name = "Shotgunner"
+	suit_store = /obj/item/gun/ballistic/shotgun/automatic/combat/shotgunlever
+	backpack_contents = list(
+		/obj/item/ammo_box/shotgun/buck = 2,
+		/obj/item/gun/ballistic/automatic/pistol/n99 = 1,
+		/obj/item/clothing/under/f13/ncr_camo = 1,
+		/obj/item/clothing/suit/armor/f13/metalarmor/steelbib = 1,
+		/obj/item/clothing/head/helmet/armyhelmet = 1,
+	)
+
+
 //NON-COMBATANTS
+//As of writing this, Scientist has been fairly active as a combatant lately and thus this is void.
```

Comment:
Scientist, regardless of what you 'might see' in game, is generally non-combatant and, as such, doesn't need combat kits.